crates/database/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "database"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+sqlx = { version = "0.6", default-features = false, features = ["postgres"] }
```

Comment:
Do you also need to enable `runtime-tokio-native-tls` feature for `sqlx`?

Otherwise _I think_ `cargo test -p database` might not work (i.e. it is working when running tests for the workspace because `orderbook` enables this feature).